src/index.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './redux/configureStore';
+// import { fetchMission } from './redux/mission/mission';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
+// store.dispatch(fetchMission);
+
```

Comment:
kindly remove comments as it may reduce the readability of your code and it could increase the time it takes for your application or website to load